src/Controls/src/Core/Handlers/Items/iOS/TemplatedCell.cs

```diff
@@ -59,6 +59,15 @@ protected void ClearConstraints()
 			ConstrainedDimension = default;
 		}
 
+		public void Unbind()
```

Comment:
@mattleibow does this make sense to be `public`? Maybe should be `internal`?

Reply:
`Bind` is public, so this was intentionally made to match.

Note that `TemplatedCell` itself is semi-public and hard to customize at the moment. There are subclasses that are internal and thus unusable, and replicating the subclassed functionality is difficult because `PlatformHandler` property is internal.